src/data.ts

```diff
@@ -0,0 +1,26 @@
+
+
```

Comment:
Something to think about in a future version is to come up with a way of compressing this data. Adding all these strings to a JS bundle is a significant size increase (est. ~60KB based on minified `data.js`). This could be done with, for example, the CompressionStream API. Let me know your throughts

Reply:
In Thorium / navigator in some cases we pass lots of textual data (serialized JSON) via URL query parameters to guarantee that the webview / iframe resolves the data "synchronously" (instant access). We use code that looks like this (there is some usage of NodeJS / Electron Buffer API but nothing that couldn't be pure Web API):

```ts
const json = { /* LOTS OF DATA */ };
const jsonStr = JSON.stringify(json);
const cs = new CompressionStream("gzip");
const csWriter = cs.writable.getWriter();
csWriter.write(new TextEncoder().encode(jsonStr));
csWriter.close();
const buff = Buffer.from(await new Response(cs.readable).arrayBuffer());
const b64 = buff.toString("base64");
// pass b64 as escaped URL query parameter
```

The inverse operation:

```ts
const buff = Buffer.from(b64, "base64");
const cs = new DecompressionStream("gzip");
const csWriter = cs.writable.getWriter();
csWriter.write(buff);
csWriter.close();
const buffer = Buffer.from(await new Response(cs.readable).arrayBuffer());
const jsonStr = new TextDecoder().decode(buffer);
const json = JSON.parse(jsonStr);
```